docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,256 @@
+---
+id: getting-started-java-spring
+title: Getting started as a Java developer using Spring
+sidebar_label: Getting started as a Java developer using Spring
+description: "Use Spring Boot and the Spring Zeebe SDK to interact with your local Self-Managed Camunda 8 installation."
+keywords: [java, spring, spring zeebe, getting started, user guide, tutorial]
+---
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--medium">1 hour</span>
+
+In this guide, we'll step through using Spring Boot and the [Spring Zeebe SDK](/apis-tools/spring-zeebe-sdk/getting-started.md) to interact with your local Self-Managed Camunda 8 installation.
+
+By the end of this tutorial, you'll be able to use Spring and Java code with Zeebe to:
+
+- Deploy a process model.
+- Initiate a process instance.
+- Handle a service task.
+
+For example, in this guide we will outline a BPMN model to receive a payment request, prepare a transaction, charge a credit card, and execute a payment:
+
+![example BPMN model to receive a payment request, prepare a transaction, charge a credit card, and execute a payment](./img/prepare-transaction-example.png)
+
+:::note
+This tutorial is not intended for production purposes.
+:::
+
+## Prerequisites
+
+Before getting started, ensure you:
+
+- Can access to your preferred code editor or IDE.
+- Have Java [installed locally](https://www.java.com/en/download/).
+- Have Docker Compose installed on your desktop. For additional guidance, visit our [Docker Compose documentation](/self-managed/setup/deploy/local/docker-compose.md) and [related GitHub respository](https://github.com/camunda/camunda-platform?tab=readme-ov-file#using-docker-compose).
+- Install [Desktop Modeler](https://camunda.com/download/modeler/).
+
+## Step 1: Install Camunda 8 Self-Managed
+
+If you haven't already, ensure you have Camunda 8 Self-Managed installed locally:
+
+1. To spin up a Camunda 8 Self-Managed environment locally, you can use the `docker-compose.yaml` file in [this repository](https://github.com/camunda/camunda-platform).
```

Comment:
```suggestion
1. Use the `docker-compose.yaml` file in [this repository](https://github.com/camunda/camunda-platform).
```
Little nit-picky, but let's make this briefer so the number list items are straightforward actions.